test/unit/style-spec/expression.test.js

```diff
@@ -19,3 +19,26 @@ test('createExpression', (t) => {
 
     t.end();
 });
+
+test('evaluate expression', (t) => {
```

Comment:
Can this be an integration test?

Reply:
It started out that way, but the expression tests use the non-error-checking `createExpression()` so they can check error messages.  Since this is a post-evaluation check specific to the error-checking path, it's not a great fit for those tests.  

🤔 We could add a _render_ test, though.